modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access

```

Comment:
This module needs a better title and anchor ID. We have many different ways to configure SSH and I am not sure any one of them is quick. This method uses the local SSH client or OpenSSH, so the title and file name need to reflect this.

I also recommended that you not use 'leverage' as a verb. It sounds jargony to me. "Use" works just as well. "Relay"? I think "send" would be simpler.